client/src/pages/Login.js

```diff
@@ -19,16 +23,63 @@ function Login() {
 		await axios
 			.post('login', { user })
 			.then(async (res) => {
-				console.log(res);
-				await localStorage.setItem('token', res.headers.get('Authorization'));
-				await localStorage.setItem(
-					'current_user',
-					JSON.stringify(res.data.data)
-				);
-				navigate('/');
+				if (res.status === 200) {
+					await localStorage.setItem('token', res.headers.get('Authorization'));
+					await localStorage.setItem(
+						'current_user',
+						JSON.stringify(res.data.data)
+					);
+					navigate('/');
```

Comment:
should refresh current ref as user may want to stay at the page before they log in